kuma/wiki/templates/wiki/base.html

```diff
@@ -10,6 +10,7 @@
     {% endif %}
 {% endblock %}
 
-{% if not scripts %}
-  {% set scripts = ('wiki',) %}
-{% endif %}
```

Comment:
Wondering about the same thing here. Was there a reason we were checking for the non-existence of `scripts`?